plugins/continuous-toolbox/package-lock.json

```diff
@@ -16,7 +16,7 @@
         "node": ">=8.17.0"
       },
       "peerDependencies": {
-        "blockly": "^11.0.0"
```

Comment:
Unexpected diff here.

Reply:
Is it possible that you did a local installation and now need to clean out your modules and reinstall?

Reply:
Done.